groups/toc/groups.yaml

```diff
@@ -0,0 +1,17 @@
+# Technical Oversight Committee
+groups:
+  - email-id: toc@knative.team
+    name: toc
+    description: |-
+      Technical Oversight Committee Email
+    settings:
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      ReconcileMembers: "true"
+    owners:
+      - markusthoemmes@knative.team
+    members:
+      - evankanderson@knative.team
+      - grantr@knative.team
```

Comment:
One naive question: are you getting these members from the existing groups? This list is outdated since grantr@ is not working on Knative anymore.